### DISCUSSION TITLE
How to make a running graph?

### DISCUSSION THREAD

**Discussion by cotonec** [Bodmer/TFT_eSPI#1950] [Q&A - General] [answered]
Good afternoon! I'm trying to figure out your examples with plotting.  I'm trying to figure out if it's possible to make a running graph (similar to a plotter) so that after filling in the lines begin to shift to free up space. Is it possible to do this? If so, how?

**Comment by Bodmer** [comment, marked as answer]
If sufficient RAM is available then a scrolling sprite can be used. See example here:
https://github.com/Bodmer/TFT_eSPI/blob/master/examples/Sprite/Sprite_scroll/Sprite_scroll.ino

**Reply by cotonec** [reply]
Thanks!

**Reply by cotonec** [reply]
I have a question. Is there a similar command (to set the start of the graph construction point) gr.drawGraph(160, 0) the same only for sprites?

UPDATE:  As far as I understand this is the command pushSprite

**Comment by Bodmer** [comment]
Not sure what you mean but setScrollRect(...) will reset the scroll window.
